apps/ecparam.c

```diff
@@ -349,47 +349,56 @@ int ecparam_main(int argc, char **argv)
         print_bignum_var(out, ec_order, "ec_order", len, buffer);
         print_bignum_var(out, ec_cofactor, "ec_cofactor", len, buffer);
         BIO_printf(out, "    int ok = 0;\n"
```

Comment:
I'd like the first string to end up as its own line as well, so it's aligned with the rest of them